AttributeError when using PyAnova.get_status

```
>       assert (await device.get_status()) == "running"
test_client.py:103: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
..\client.py:513: in get_status
    self._last_status = await self.send_read_command(ReadCommands.Status)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
cls = <enum 'ReadCommands'>, name = 'Status'
    def __getattr__(cls, name):
        """
        Return the enum member matching `name`
    
        We use __getattr__ instead of descriptors or inserting into the enum
        class' __dict__ in order to support `name` and `value` being both
        properties for enum members (which live in the class' __dict__) and
        enum members themselves.
        """
        if _is_dunder(name):
            raise AttributeError(name)
        try:
            return cls._member_map_[name]
        except KeyError:
>           raise AttributeError(name) from None
E           AttributeError: Status
C:\Program Files\Python311\Lib\enum.py:783: AttributeError
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -5,6 +5,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.2.#] - To be released
+- Fix AttributeError on ``PyAnova.get_status()`` (#17)
+
 ## [0.2.3] - 2024-11-11
 - Update to protobuf-5 (#15)
 

--- a/pyanova_nano/client.py
+++ b/pyanova_nano/client.py
@@ -110,7 +110,7 @@ class PyAnova:
         return self._device
 
     @property
-    def last_status(self) -> SensorValues:
+    def last_status(self) -> Optional[SensorValues]:
         """Return the last polled sensor values."""
         return self._last_sensor_values
 

--- a/pyanova_nano/client.py
+++ b/pyanova_nano/client.py
@@ -508,11 +508,17 @@ class PyAnova:
 
         return sensor_values
 
-    async def get_status(self) -> str:
-        """Return the current device status (either stopped or running)."""
-        self._last_status = await self.send_read_command(ReadCommands.Status)
-        print(self._last_status)
-        return "stopped" if self._last_sensor_values.motor_speed == 0 else "running"
+    async def get_status(self, refresh: bool = True) -> str:
+        """Return the current device status (either stopped or running).
+
+        Args:
+            refresh: If true,
+        """
+        if refresh:
+            await self.get_sensor_values()
+
+        is_running = self.last_status and self.last_status.motor_speed != 0
+        return "running" if is_running else "stopped"
 
     async def get_current_temperature(self) -> float:
         """Return the current temperature."""

